src/main/resources/application.yaml

```diff
@@ -10,7 +10,7 @@ idam:
     useStub: false
     url: ${S2S_URL}
     secret: ${S2S_SECRET}
-    microservice: ${REFORM_SERVICE_NAME:jobscheduler}
+    microservice: jobscheduler
```

Comment:
terraform still sets this variable in `main.tf`:
```
REFORM_SERVICE_NAME = "${var.microservice}"
```

Reply:
the logging library needs it. I could just rename the app to `jobscheduler` rather than `job-scheduler` but will mean binning all of the infra deployed so far and redeploying i think. not the end of the world but the naming convention gets a little spoiled.
This basically comes down to the s2s implementation being tightly coupled to ansible and im worried a key with a hyphen in it in ansible might break?